NAMESPACE

```diff
@@ -477,6 +478,7 @@ export(stat_summary_bin)
 export(stat_summary_hex)
 export(stat_unique)
 export(stat_ydensity)
+export(stop_server)
```

Comment:
two different spellings of server, in `start_servr` and `stop_server`, is potentially confusing. please change to `stop_servr` for consistency.